Sources/MapboxNavigation/NavigationMapViewIdentifiers.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+struct IdentifierString {
```

Comment:
Split this struct into `NavigationMapView.LayerIdentifier`, `NavigationMapView.SourceIdentifier`, and `NavigationMapView.ImageIdentifier`. That’ll simplify the naming of many of these constants and avoid bugs where a layer identifier is used as a source identifier or vice versa.